lib/plugins.js

```diff
@@ -252,10 +254,11 @@ function plugins (settings, deviceId) {
         const cryptoCodesCount = cryptoCodes.length
         const tickers = arr.slice(4, cryptoCodesCount + 4)
         const balances = arr.slice(cryptoCodesCount + 4, 2 * cryptoCodesCount + 4)
-        const testNets = arr.slice(2 * cryptoCodesCount + 4, arr.length - 1)
+        const testNets = arr.slice(2 * cryptoCodesCount + 4, arr.length - 2)
         const coinParams = _.zip(cryptoCodes, testNets)
         const coinsWithoutRate = _.map(mapCoinSettings, coinParams)
-        const areThereAvailablePromoCodes = arr[arr.length - 1] > 0
+        const areThereAvailablePromoCodes = arr[arr.length - 2] > 0
+        const individualDiscounts = arr[arr.length - 1]
```

Comment:
We should not send individualDiscounts for all customers, can't we just add it to the customer response?

Reply:
You're completely correct. Definitely an oversight on my part, I'll work on it